lms/djangoapps/verify_student/services.py

```diff
@@ -56,3 +60,36 @@ def start_verification(self, course_id, related_assessment, item_id):
             )
         )
         return re_verification_link
+
+    def skip_verification(self, checkpoint_name, user_id, course_id):
+        """Create the add verification attempt
+
+        Args:
+            course_id(str): A string of course_id
+            user_id(str): User Id string
+            checkpoint_name(str): Verification checkpoint name
+
+        Returns:
+            None
+        """
+        course_key = CourseKey.from_string(course_id)
+        checkpoint = VerificationCheckpoint.objects.get(course_id=course_key, checkpoint_name=checkpoint_name)
+
+        # if user do not already skipped the attempt for this course only then he can skip
+        try:
+            SkippedReverification.add_skipped_reverification_attempt(checkpoint, user_id, course_key)
+        except IntegrityError:
+            log.exception("Skipped attempt already exists for user %s: with course %s:", user_id, unicode(course_id))
+
+    def is_user_skipped_reverification(self, user_id, course_id):
```

Comment:
Would it make sense to do this check in `get_status()`, and return "skipped" if the user has skipped?